modules/apps/27-interchain-accounts/controller/keeper/keeper.go

```diff
@@ -101,7 +102,7 @@ func (k Keeper) ClaimCapability(ctx sdk.Context, cap *capabilitytypes.Capability
 	return k.scopedKeeper.ClaimCapability(ctx, cap, name)
 }
 
-// GetActiveChannelID retrieves the active channelID from the store keyed by the provided portID
+// GetActiveChannelID retrieves the active channelID from the store key by the provided portID
```

Comment:
Think this comment should stay the same, but probably add a comma so it reads correctly.

```suggestion
// GetActiveChannelID retrieves the active channelID from the store, keyed by the provided portID
```

Reply:
Good catch, I've updated in in two places.